chaos_wrapper.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+function rollback() {
+    # avoid commits from the same faulty PR
+    rb_commit=$(git log --author Chaos --format=format:%H | tail -n +2 | head -n 1)
```

Comment:
TBH, I considered creating such a commit (you can imagine what I might have put in it) :stuck_out_tongue: